assembl/views/__init__.py

```diff
@@ -253,7 +324,8 @@ def process_export_list(ls):
         raven_url=config.get('raven_url') or '',
         activate_tour=str(config.get('activate_tour') or False).lower(),
         providers=json.dumps(providers),
-        translations=codecs.open(jedfilename, encoding='utf-8').read()
+        translations=codecs.open(jedfilename, encoding='utf-8').read(),
```

Comment:
FYI, codecs.open is quite old, in Python 2.7, you can now import open from the io package (backported from python 3.) 
```
from io import open
```
and it supports the encoding param. The builtin open function doesn't support encoding indeed.